rmf_visualization_schedule/src/ScheduleMarkerPublisher.hpp

```diff
@@ -116,7 +116,7 @@ class ScheduleMarkerPublisher : public rclcpp::Node
     auto sub_param_opt = rclcpp::SubscriptionOptions();
     sub_param_opt.callback_group = _cb_group_param_sub;
     _param_sub = this->create_subscription<RvizParamMsg>(
-      node_name + "/param",
+      "rviz_node/param",
```

Comment:
Good catch. Can we rename this topic to `rmf_visualization_schedule/parameters`?
You will need to make the change here too https://github.com/open-rmf/rmf_visualization/blob/56765c8c5a679dbf3f2e5ad172e3b25c604079d8/rmf_visualization_rviz2_plugins/src/SchedulePanel.cpp#L36

Reply:
Yes i can change this. But just a heads up, this change will also require a change in `rmf_demos` `.rviz` file. (Users will also need to change their own .rviz). This should be alright?

Reply:
Actually let's rename it `rmf_visualization/parameters` as it is also used in `rmf_visualization_building_systems`.
So in addition, we'll need to change the topic name here https://github.com/open-rmf/rmf_visualization/blob/56765c8c5a679dbf3f2e5ad172e3b25c604079d8/rmf_visualization_building_systems/rmf_visualization_building_systems/rmf_visualization_building_systems.py#L66
And yes, we'll need to update `.rviz` files in `rmf_demos`.